### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by sschuste**

### CODE REVIEW

File: ui/src/i18n/de.json [hunk 1]
```diff
@@ -103,11 +103,11 @@
...
-    "logs.message_download_warning": "WARNING: Homebridge log files may contain personal information, passwords or access tokens. Please review the log file and remove any sensitive information before sharing with others.",
-    "logs.message_log_file_truncated": "Log file truncated.",
-    "logs.message_truncate_log_warning": "This action will irreversibly delete your existing Homebridge logs. Are you sure you want to continue?",
-    "logs.title_download_log_file": "Download Log File",
-    "logs.title_truncate_log_file": "Truncate Log File",
+    "logs.message_download_warning": "ACHTUNG: Homebridge-Logdateien können persönliche Informationen, Passwörter or Zugrifftokens beinhalten. Bitte sieh dir die Logdatei an und entferne alle sensitiven Informationen, bevor du sie mit anderen teilst.",
+    "logs.message_log_file_truncated": "Logdatei abgeschnitten.",
+    "logs.message_truncate_log_warning": "Durch diese Aktion werden deine bestehenden Homebridge-Logdateien unwiderruflich gelöscht. Bist du sicher, dass du fortfahren willst?",
+    "logs.title_download_log_file": "Logdatei herunterladen",
+    "logs.title_truncate_log_file": "Logdatei abscheiden",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:110]: There is a misspelling. Should be `Logdatei abschneiden`

File: ui/src/i18n/de.json [hunk 2]
```diff
@@ -103,11 +103,11 @@
     "login.title_login": "Anmeldung",
     "logs.label_download": "Download",
     "logs.label_truncate": "Truncate",
-    "logs.message_download_warning": "WARNING: Homebridge log files may contain personal information, passwords or access tokens. Please review the log file and remove any sensitive information before sharing with others.",
-    "logs.message_log_file_truncated": "Log file truncated.",
-    "logs.message_truncate_log_warning": "This action will irreversibly delete your existing Homebridge logs. Are you sure you want to continue?",
-    "logs.title_download_log_file": "Download Log File",
-    "logs.title_truncate_log_file": "Truncate Log File",
+    "logs.message_download_warning": "ACHTUNG: Homebridge-Logdateien können persönliche Informationen, Passwörter or Zugrifftokens beinhalten. Bitte sieh dir die Logdatei an und entferne alle sensitiven Informationen, bevor du sie mit anderen teilst.",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:106]: `or` should be translated as well to `oder` maybe `und/oder` would fit even better.

File: ui/src/i18n/de.json [hunk 3]
```diff
@@ -62,9 +62,9 @@
...
+    "backup.message_restore_help_one": "Stelle ein Backup-Archiv, welches zuvor mit Homebridge Config UI X erstellt wurde, wieder her. Deine Homebridge-Konfiguration, Geräte im Cache, HomeKit-Verbindungen und Homebridge Config UI X-Benutzerkonten werden wiederhergestellt. Auch die Homebridge-Plugins, die Du installiert hattest, werden über die npm Registry heruntergeladen.",
     "backup.message_restore_help_two": "Du musst dich nach der Wiederherstellung mit den Anmeldedaten aus der wiederhergestellten Instanz bei Homebridge Config UI X anmelden.",
-    "backup.message_restore_warning": "Das Wiederherstellen von einem Backup wird unwiderruflich die aktuelle Homebridge-Konfiguration überschreiben.",
+    "backup.message_restore_warning": "Durch das Wiederherstellen von einem Backup wird die aktuelle Homebridge-Konfiguration unwiderruflich überschreiben.",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:67]: `überschreiben` should be `überschrieben`

File: ui/src/i18n/de.json [hunk 4]
```diff
@@ -204,23 +204,23 @@
     "reset.message_all_automations_will_be_reset": "Alle Automatisierungen müssen nach einem Reset neu konfiguriert werden.",
     "reset.message_force_restart_hb_help_text": "Benutze diese Aktion, damit der hb-service Dienst einen vollständigen Neustart erzwingt.",
     "reset.message_need_to_remove_homebridge_accessory_from_home_app": "Du musst das vorhandene Homebridge-Geräte manuell aus der Home-App entfernen.",
-    "reset.message_remove_cached_accessories": "Diese Aktion wird alle gecachten Geräte von deiner Homebridge-Instanz entfernen..",
+    "reset.message_remove_cached_accessories": "Diese Aktion wird alle Geräte aus dem Cache deiner Homebridge-Instanz entfernen.",
     "reset.message_remove_cached_accessories_single": "Lösche ein einzelnes Gerät aus dem Geräte-Cache.",
-    "reset.message_remove_cached_accessories_single_warning": "Mit Vorsicht verwenden. Das Entfernen bestimmter gecachter Geräte kann unbeabsichtigte Folgen haben.",
+    "reset.message_remove_cached_accessories_single_warning": "Mit Vorsicht verwenden. Das Entfernen bestimmter Geräte vom Cache kann unbeabsichtigte Folgen haben.",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:209]: Should be `aus dem Cache`

File: ui/src/i18n/de.json [hunk 5]
```diff
@@ -204,23 +204,23 @@
...
-    "reset.title_clear_cached_accessories": "Gecachte Geräte entfernen",
-    "reset.title_clear_cached_accessories_single": "Entfernen Sie ein einzelnes gecachtes Gerät",
+    "reset.title_clear_cached_accessories": "Geräte aus dem Cache entfernen",
+    "reset.title_clear_cached_accessories_single": "Entferne ein einzelnes Gerät aus dem Cache",
     "reset.title_force_restart_hb_service": "Erzwinge Neustart des Dienstes",
     "reset.title_force_restart_hb_service_short": "Dienst neustarten",
     "reset.title_reset": "Zurücksetzen",
     "reset.title_reset_homebridge_accessory": "Zurücksetzen der Homebridge-Geräte",
-    "reset.title_unpair_bridges_cameras_tv_external_accessories": "Entkoppeln Sie Bridges / Kameras / Fernseher / externes Zubehör",
+    "reset.title_unpair_bridges_cameras_tv_external_accessories": "Entkoppele Bridges / Kameras / Fernseher / externes Zubehör",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:220]: Should be `Entkopple`

File: ui/src/i18n/de.json [hunk 6]
```diff
@@ -232,11 +232,11 @@
...
-    "settings.reset.help": "Wenn Sie Probleme beim Koppeln von Homebridge oder einem externem Gerät wie einer Kamera oder einem Fernseher mit HomeKit haben, müssen Sie möglicherweise das Gerät entkoppeln oder Ihre Homebridge-Instanz zurücksetzen.",
+    "settings.manage_cache.help": "Wenn ein nicht mehr vorhandenes Gerät noch in HomeKit angezeigt wird, musst du es möglicherweise aus dem Cache entfernen. Du kannst entweder ein einzelnes Gerät oder alle zwischengespeicherten Geräte aus dem Cache entfernen.",
+    "settings.reset.help": "Wenn du Probleme beim Koppeln von Homebridge oder einem externem Gerät wie einer Kamera oder einem Fernseher mit HomeKit hast, musst du möglicherweise das Gerät entkoppeln oder deine Homebridge-Instanz zurücksetzen.",
     "settings.title_actions": "Aktionen",
     "settings.title_environment_variables": "Umgebungsvariablen",
-    "settings.title_manage_cached_accessories": "Verwalten von gecachten Geräten",
+    "settings.title_manage_cached_accessories": "Verwalten von Geräten im Cache",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:239]: In my opinion this translation isn't better than the one before.
I would not translate it this way.

**Review by TautFlorian — CHANGES_REQUESTED** [review]
There are some misspellings and my opinions about some of the changes you made. 
Would appreciate to hear your thoughts on this.

File: ui/src/i18n/de.json [hunk 7]
```diff
@@ -103,11 +103,11 @@
     "login.title_login": "Anmeldung",
     "logs.label_download": "Download",
     "logs.label_truncate": "Truncate",
-    "logs.message_download_warning": "WARNING: Homebridge log files may contain personal information, passwords or access tokens. Please review the log file and remove any sensitive information before sharing with others.",
-    "logs.message_log_file_truncated": "Log file truncated.",
-    "logs.message_truncate_log_warning": "This action will irreversibly delete your existing Homebridge logs. Are you sure you want to continue?",
-    "logs.title_download_log_file": "Download Log File",
-    "logs.title_truncate_log_file": "Truncate Log File",
+    "logs.message_download_warning": "ACHTUNG: Homebridge-Logdateien können persönliche Informationen, Passwörter or Zugrifftokens beinhalten. Bitte sieh dir die Logdatei an und entferne alle sensitiven Informationen, bevor du sie mit anderen teilst.",
+    "logs.message_log_file_truncated": "Logdatei abgeschnitten.",
```
Comment (TautFlorian) [inline ui/src/i18n/de.json:107]: Maybe we should talk generally about dots. In this case there is a dot added to the line but in most cases there is none.
We should decide for one option, in my eyes we shouldn't add a line ending point.

**Review by TautFlorian — CHANGES_REQUESTED** [review]

**Comment by oznu** [comment]
@sschuste - I'll wait for your comment on the above before merging 😄 

Thanks everyone for contributing!

**Comment by TautFlorian** [comment]
@oznu  I'm going to create a pull request by my own fixing `de.json` this evening because @sschuste seems not to be very active.

**Comment by oznu** [comment]
I'll merge this. @TautFlorian - feel free to do another PR with any changes you think improve the translation further 😄 

Thanks @sschuste

**Pull request merged**